src/sentry/static/sentry/app/views/performance/landing/content.tsx

```diff
@@ -128,7 +128,7 @@ class LandingContent extends React.Component<Props, State> {
               [...eventView.fields, {field: 'tps()'}],
               ['epm()', 'eps()']
             )}
-            omitTags={[`user_misery_prototype(${organization.apdexThreshold})`]}
+            omitTags={[`count_miserable(user, ${organization.apdexThreshold})`]}
```

Comment:
Is this still necessary? Can we leave it exposed if the user chose to use it?

Reply:
yep, removed since we are exposing this as an aggregate field in discover